src/main/java/Duke.java

```diff
@@ -131,6 +160,9 @@ public static void addTaskIn(Task task,ArrayList<Task> tasks) {
     }
 
     public static void list (ArrayList<Task> tasks){
+        if(tasks.size() == 0){
+            throw new RuntimeException();
+        }
         System.out.println("    ____________________________________________________________");
         for (int i = 0; i < tasks.size(); i++) {
             System.out.print("     "+(i+1)+"."+"["+tasks.get(i).getType()+"]"+"["+ tasks.get(i).getStatus()+"] "+tasks.get(i).getTaskName()+" ");
```

Comment:
A very long line, consider breaking into 2 lines or more. The concatenation seem a bit excessive as well. Consider combining the hard-coded parts together instead of concatenating. E.g.: `"     " + (i + 1) + ".["`